benchmarks/templates/benchmarks/tutorials/models/deepdive_1.html

```diff
@@ -41,87 +40,70 @@ <h3 class="benefits_heading is-size-3-mobile">Let's Dive Deeper!</h3>
-        this case <span class="special_format">/models</span>. This <span class="special_format">/models</span> folder then
-        contains a folder called <span class="special_format"> /resnet50_tutorial</span>.
-        This final also contains 4 other Python files. The overall structure of the submission
-        package is shown below:
+        The correct overall structure for a model submission package is shown in the code inset below.  The root folder
+        can be named anything (in this case it's called "packaging_tutorial").  The root folder contains another folder
+        for the plugin itself, in this case <span class="special_format">/models</span>.
+        This <span class="special_format">/models</span> folder then contains a folder called
+        <span class="special_format"> /resnet50_tutorial</span>.
     </p>
-    <img class="submission example" width="500" height="auto" src="{% static "/benchmarks/img/submission_structure.png" %}" />
-    <p class="benefits_info is-size-5-mobile shorter">
-        Just like the sample submission, each submission that you submit must contain at least the 2 required
-        Python files: <span class="special_format">__init__.py</span> and <span class="special_format">test.py</span>.
-        The other two files, <span class="special_format">setup.py</span> and <span class="special_format">model.py</span>,
-        are optional, but will be explored below along with the two required files. Let's explore further with the first required file,
+        The <a href="https://github.com/brain-score/vision/tree/master/brainscore_vision/models/resnet50_tutorial">resnet50_tutorial folder</a> in the repo
+        is an example of a properly formatted submission folder; it contains 3 Python files and a single .txt file.
+        Just like this example, all submissions must contain at least three Python files:
+        <span class="special_format">__init__.py</span>, <span class="special_format">test.py</span>,
+        and <span class="special_format">model.py</span> (these are required).
+        The other file: <span class="special_format">requirements.txt</span>, is optional but will be examined below
+        along with the required files. Let's explore further with the first required file,
         <span class="special_format">__init__.py</span> file.
     </p>
-    <p class="benefits_info is-size-5-mobile shorter is-italic">
-        Note: this would also be the same overall structure as your submission
-        package, or if you are submitting any other plugins (benchmarks, metrics, or data). Feel free
-        to base your future submissions off of this package, just please do not submit the
-    <span class="special_format">resnet50_tutorial</span> model.
-    </p>
+    <img class="submission example" width="500" height="auto" src="{% static "/benchmarks/img/submission_structure.png" %}" />
```

Comment:
Picture needs to be updated to exclude `setup.py` and include `requirements.txt`.

Reply:
uploaded a new image.  changed the image name to submissions_structure2.png

Reply:
i updated the image.